### ISSUE TITLE
Import fails after fixing style

### ISSUE THREAD

**Issue by ercasta** [CoderDojoBrianza/CoderDojo-Mobile-Toolbox#5]
@sandrobonazzola , the stile fix is causing issues.

in `admin.py`, the change from:
```
from .models import *
```

to 
```
import models
```

is causing a runtime error at startup:

```
  File "(my folder hidden here)\coderdojomobile\admin.py", line 4, in <module>
    import models
ModuleNotFoundError: No module named 'models'
```

Maybe PYTHONPATH needs to be changed as a result of the style change?

**Comment by mdibit** [comment]
I think so. Models is in the same dir as the other .py files, isn't it? I
think you must add This dir to path or just leave .models

Il sab 4 ago 2018, 12:37 ercasta <notifications@github.com> ha scritto:

**Comment by ercasta** [comment]
I have lots of issues with imports after the style fix. As I am currently unable to solve them, I'll go on with development on my branch, then I think I'll need your help to fix them

**Comment by sandrobonazzola** [comment]
Fixed by PR #6
